chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -36,6 +39,18 @@
 logger = logging.getLogger(__name__)
 
 
+def move_to_recycle_bin(retry_state: RetryCallState) -> None:
+    """Move the index directory to the wastebin"""
+    data_path = retry_state.args[0]
+    if os.path.exists(data_path):
+        shutil.move(data_path, "C:\\$Recycle.Bin")
```

Comment:
C may not exist

Reply:
I added a more generic handler for deletion using ctypes.

Reply:
Do we want to move this to utils?

Reply:
Nice, yeah that sounds good to me, thanks